Src/BootCamp.Chapter/Checks.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
+
+class Lesson3
+{
+	public static void Demo()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4859834)